.eslintrc.json

```diff
@@ -56,55 +51,55 @@
     "require-atomic-updates": "off",
     "sort-keys": "off",
 
-    "node/handle-callback-err": "error",
-    "node/no-callback-literal": "error",
-    "node/no-exports-assign": "error",
-    "node/no-extraneous-import": "error",
-    "node/no-extraneous-require": "error",
-    "node/no-missing-import": ["error", {
+    "n/handle-callback-err": "error",
+    "n/no-callback-literal": "error",
+    "n/no-exports-assign": "error",
+    "n/no-extraneous-import": "error",
+    "n/no-extraneous-require": "error",
+    "n/no-missing-import": ["error", {
       "allowModules": [
         "chalk"
       ]
     }],
-    "node/no-missing-require": ["error", {
+    "n/no-missing-require": ["error", {
       "allowModules": [
         "ava"
       ]
     }],
-    "node/no-new-require": "error",
-    "node/no-path-concat": "error",
-    "node/no-process-exit": "error",
-    "node/no-unpublished-bin": "error",
-    "node/no-unpublished-import": "error",
-    "node/no-unpublished-require": "off",
-    "node/no-unsupported-features/es-builtins": "error",
-    "node/no-unsupported-features/es-syntax": "error",
-    "node/no-unsupported-features/node-builtins": ["error", {
+    "n/no-new-require": "error",
+    "n/no-path-concat": "error",
+    "n/no-process-exit": "error",
+    "n/no-unpublished-bin": "error",
+    "n/no-unpublished-import": "error",
+    "n/no-unpublished-require": "error",
+    "n/no-unsupported-features/es-builtins": "error",
+    "n/no-unsupported-features/es-syntax": "error",
+    "n/no-unsupported-features/node-builtins": ["error", {
       "ignores": [
         "fs.promises"
```

Comment:
ONLY if it's as simple for you as deleting the "ignores" section, is it still necessary to customize this rule, or is `["error"]` enough with the latest `package.json` version changes?